doc/cookbook/source/examples/clustering/gmm.rst

```diff
@@ -0,0 +1,49 @@
+=======================
+Gaussian Mixture Models
+=======================
+
+A Gaussian Mixture Model is a probabilistic model that assumes all the data points are generated from a mixture of a finite number of Gaussian distributions with unknown parameters.
+
+.. math::
+
+    p(\theta|\boldsymbol{X}) = \sum_{i=1}^{K}{\phi_i \cal{N}(\boldsymbol{\mu_i}, \boldsymbol{\Sigma_i})}
+
+where :math:`p(\theta|\boldsymbol{X})` is posterior distribution, :math:`K` denotes number of mixture components, :math:`\phi_i` denotes weight for :math:`i`-th component, :math:`\cal{N}` denotes a multivariate normal distribution with mean vector :math:`\boldsymbol{\mu_i}` and covariance matrix :math:`\boldsymbol{\Sigma_i}`.
+
+Expectation Maximization(EM) algorithm is used to train the model iteratively until the local maxima in the model's likelihood is obtained.
+
+See Chapter 20 in :cite:`barber2012bayesian` for a detailed introduction.
+
+-------
+Example
+-------
+
+Start by creating CDenseFeatures (here 64 bit floats aka RealFeatures) as
+
+.. sgexample:: gmm.sg:create_features
+
+Initialize :sgclass:`GMM` by deciding the number of components for the mixture model.
+
+.. sgexample:: gmm.sg:create_gmm_instance
+
+Provide training features to the :sgclass:`GMM` object and train it by using EM algorithm. Split-Merge Expectation-Maximization algorithm :cite:`ueda2000smem` can also be used for training which can be found in :sgclass:`GMM` documentation. After training, one can sample data-points from the trained model.
```

Comment:
Please mention the split merge version at the end, the first part should just be standard EM